framework/include/mediaplayer/mediaplayer.h

```diff
@@ -0,0 +1,263 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __MEDIAPLAYER_H__
+#define __MEDIAPLAYER_H__
+
+#include <queue.h>
+#include <sys/types.h>
+
+enum media_result_e {
+	MEDIA_OK,		
+	MEDIA_ERROR,
+	MEDIA_ERROR_THREAD_INIT,
+	MEDIA_ERROR_PCM_WRITE,
+	MEDIA_ERROR_WRITE_TO_FILE,
```

Comment:
@shivgarg @btheosam  WRITE_TO_FILE, WRITE_ON_FILE what is correct?

Reply:
In my opinion WRITE_TO_FILE is correct.